docs/api.md

```diff
@@ -220,7 +220,7 @@ Object (also known as a "hash") of application metadata to embed into the execut
 
 ### `err`
 
-*Error* (or *Array*, in the case of an `ncp` error)
+*Error* (or *Array*, in the case of an `copy` error)
```

Comment:
We should probably update all of the `copy` calls to use that (in a separate PR). Having `err` be potentially more than one type is problematic for users.